content/en/docs/home/contribute/localization.md

```diff
@@ -38,12 +38,17 @@ Source files reside in the `/docs/` directory.
 
 ### Repository
 
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-cn](https://github.com/kubernetes/kubernetes-docs-cn).
+A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-zh](https://github.com/kubernetes/kubernetes-docs-zh).
```

Comment:
no need to change url link, but ok in here

Reply:
We are standardizing on the two letter language codes instead of the country codes.